scripts/biomedical/VirtualMetabolicHuman/README.md

```diff
@@ -0,0 +1,109 @@
+
+### Schema Overview
+
+The schema representing reaction, metabolite and microbiome data from VMH is defined in [VMH.mcf](https://raw.githubusercontent.com/suhana13/ISB-project/main/combined_list.mcf) and [VMH_enum.mcf](https://raw.githubusercontent.com/suhana13/ISB-project/main/combined_list_enum.mcf).
+
+The imported data, contains several instances of entities "Metabolite", "ChemicalReaction", "memberOfMetabolicReactionSubsystem", and "Microbe". 
+
+The Metabolite entity is used to describe chemical substances which are intermediates or end products in a metabolic reaction. It has several text value properties, representing "abbreviation", "chemicalName", "chemicalFormula", "chemicalCharge", "averageMolecularWeight", "monoIsotropicWeight", "keggID", "pubChemID", "chebiID", "humanMetabolomeDatabaseID", "parkinsonsDiseaseMapName", "reconMap", "reconMap3", "foodb", "chemSpiderID", "bioCyc", "biggID", "wikipediaNameReference", "drugBankID", "seedID", "metaNetXID", "knapsack", "metlin", "casRegistry", "epaId", "inChIKey", "inCHIString", "simplifiedMolecularInputLineEntrySystem"
+
+The ChemicalReaction entity describes chemical reactions. It has several text value properties, including "abbreviation", "reactionDescription", "chemicalEquation", "enzymeCommissionNumber", "keggReactionID", "keggOrthologyID", "cogID", "seed", "metaNetXID". The "memberOfMetabolicReactionSubsystem" returns the "MetabolicReactionSubsystem" property whose value in turn, is a text representing theh biochemical subsystem in which the reaction occurs.
+
+The Microbe entity describes the microscopic organims in the biological world. It has several text value properties including "organismTaxonomicPhylum", "organismTaxonomicClass", "organismTaxonomicOrder", "organismTaxonomicFamily", "organismTaxonomicGenus", "dataDraftCreator", "dateOfDraftCreation", "integratedMicrobialGenomeID", "ncbiID", "kBaseID". It also has properties with quantity values like "microbeGeneInteractionMetric", "microbeReactionMetric", "microbeMetaboliteMetric", "microbePhenotypeMetric". Lastly, "OxygenRequirementStatus", "MicrobialMetabolismType", "BacteriaGramStainType", "PathogenMethodOfInvasion", are of type enumeration.
+
+
+### Notes and Caveats
+
+The identifiers for a lot of databases (like KEGG, metaNetX), were missing in the dataset. So, the dcids couldnt be based off of them, and thus, chemical names were used for the rest of them. 
+The runtime for the python scripts maybe more because the downloaded dataset had to be mapped to other datasets, for extraction of required information.
+
+
+### License
+
+This data is under a [Creative Commons CC0 license](https://creativecommons.org/publicdomain/zero/1.0/). 
+
+
+## About the import
+
+### Artifacts
+
+#### Scripts
+
+`format_metabolite.py`
+`format_reaction.py`
+`format_reaction_subsystem.py`
+`format_microbes.py`
+
+
+## Examples
```

Comment:
Like with [Khoa's PR](https://github.com/datacommonsorg/data/pull/447), it might be useful to have a wrapper shell script that just runs everything with the right input args